### PR TITLE
Task Completion PRL issue EXUI-1793

### PR DESCRIPTION
**Pull request by andywilkinshmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EXUI-1793

### Change description ###

remove isComplete route parameter. Doesn't seem to do anything useful.
Move code that removes the taskToComplete parameter to only do it when the taskToComplete code might then be reeded

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by RiteshHMCTS — APPROVED** [review]

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/services/work-allocation.service.ts [hunk 1]
```diff
@@ -98,6 +102,7 @@ export class WorkAllocationService {
     if (!this.isWAEnabled()) {
       return of(null);
     }
+    console.log(`completeTask: completing ${taskId}`);
```
Comment (OgunyemiO) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/services/work-allocation.service.ts:105]: There are quite a few console logs. Do we really need them?

**Review by connorpgpmcelroy — DISMISSED** [review]
Approving - assuming console.logs for testing purposes

**Review by OgunyemiO — DISMISSED** [review]
Approve with minor comments.

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/services/work-allocation.service.ts (hunk 1 above)
Comment (andywilkinshmcts) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/services/work-allocation.service.ts:105]: I want to leave them in for now. I've actually removed quite a lot that were previously there

**Pull request merged**